components/formats-bsd/src/loci/formats/FilePattern.java

```diff
@@ -723,7 +897,11 @@ private void buildFiles(String prefix, int ndx, List<String> fileList) {
     return last;
   }
 
-  /* @deprecated */
+  /**
+   * Gets the STEP element for all blocks.
+   *
+   * @return an array containing all STEP elements.
+   */
```

Comment:
Just to clarify - your intention was to un-deprecate these three methods?  I'm fine with that given that they're still being used, just want to make sure it isn't accidental before merging.